docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,468 @@
+Continuous Patching Workflow in Azure Container Registry
+========================================================
+
+## Introduction
+
+Continuous Patching is a feature in Azure Container Registry that allows you to recurringly scan and patch specified artifacts for only OS-level  vulnerabilities within the registry. The workflow enables to set a cadence for run and create a recurring ACR task that scans your configured list of images for vulnerabilities (CVEs) using [Trivy](https://trivy.dev/) and patch them using [Copa](https://project-copacetic.github.io/copacetic/website/).
+ 
+
+> **NOTE:**
+> Continuous Patching is a limited pilot program as of October 2024.
+
+
+## Use Cases
+
+Here are a few scenarios to use Continuous Patching:
+
+- **Enforcing container security and hygiene:** Continuous Patching enables users to quickly fix OS container CVEs without the need to fully rebuild from upstream.
+- **Speed of Use:** Continuous Patching removes the dependency on upstream updates for specific images by updating packages automatically. Vulnerabilities can appear every day, while popular image publishers like Ubuntu only offer a new release once a month. With Continuous Patching, one can ensure that their image is OS vulnerability-free more frequently.
```

Comment:
`With Continuous Patching, you can ensure that container images within your registry are patched as soon as the newest set of OS vulnerabilities are detected.`